### ISSUE TITLE
Pre-trained checkpoints

### ISSUE THREAD

**Issue by agoodge** [qq456cvb/Point-Transformers#40]
Thanks for the repo, 

Are there pre-trained model checkpoints available?

Thanks

**Comment by aldinorizaldy** [comment]
I'm training Hengshuang's model now, but only with ModelNet40 and S3DIS data. I'll let you know when it's done.

**Comment by Sottix99** [comment]
> Thanks for the repo,
> 
> Are there pre-trained model checkpoints available?
> 
> Thanks

I am looking for the same thing, I even wrote to the authors of the paper but they didn't answer me

**Comment by aldinorizaldy** [comment]
> > Thanks for the repo,
> > Are there pre-trained model checkpoints available?
> > Thanks
> 
> I am looking for the same thing, I even wrote to the authors of the paper but they didn't answer me

Which model do you need? I think I have the pre-trained of Hengshuang's model on ModelNet.

Otherwise you can check the original implementation of Point Transformer and some other great models here: https://github.com/Pointcept

**Comment by Sottix99** [comment]
> > > Thanks for the repo,
> > > Are there pre-trained model checkpoints available?
> > > Thanks
> > 
> > 
> > I am looking for the same thing, I even wrote to the authors of the paper but they didn't answer me
> 
> Which model do you need? I think I have the pre-trained of Hengshuang's model on ModelNet.
> 
> Otherwise you can check the original implementation of Point Transformer and some other great models here: https://github.com/Pointcept

i would need it for my thesis. since it is expensive to train such a model from scratch, if i had some chekpoints i could do finetuning from those. I would be very grateful if you could send me the pre-trained of Hengshuang's model

**Comment by aldinorizaldy** [comment]
Please download here:
https://drive.google.com/file/d/1WxzsRYmhXQDCc4ctLGeVUM0ReBOi9bIC/view?usp=sharing
I trained the model for 180 epochs (at that time I miscalculated the training time, so I only got about 180 epochs before my machine shutting down). The instance accuracy is 90.16. 
The reported accuracy in this repo is 91.7 with 200 epochs. So, today I continued the training but could not achieve higher accuracy. Please let me know if this is sufficient for you. If not, I can give another try by training from scratch, hopefuly to get the similar accuracy.

**Comment by qq456cvb** [comment]
I don't have pretrained checkpoints to share, sorry. The models are meant to be trained from scratch, and it's pretty light. `python train_cls.py model=Hengshuang` saves the best checkpoint to `log/cls/<model>/best_model.pth`. Ping me if you run into trouble training.
